tests/e2e-openshift/gateway/00-assert.yaml

```diff
@@ -147,9 +167,12 @@ spec:
           - --log.level=info
           - --tls.server.cert-file=/etc/tempo-gateway/serving-certs/tls.crt
           - --tls.server.key-file=/etc/tempo-gateway/serving-certs/tls.key
-          image: quay.io/observatorium/api:main-2023-02-09-v0.1.2-329-g1ff4f11
+          - --tls.internal.server.cert-file=/etc/tempo-gateway/serving-certs/tls.crt
+          - --tls.internal.server.key-file=/etc/tempo-gateway/serving-certs/tls.key
+          - --tls.healthchecks.server-ca-file=/etc/tempo-gateway/cabundle/service-ca.crt
+          - --tls.healthchecks.server-name=tempo-simplest-gateway.kuttl-ocp-gateway.svc.cluster.local
```

Comment:
because of this I have used the `--namespace` with kuttl. It seems the simplest solution for now.